chain/messagepool/selection_test.go

```diff
@@ -936,7 +936,7 @@ func testCompetitiveMessageSelection(t *testing.T, rng *rand.Rand) (float64, flo
 
 	capacityBoost := 0.0
 	rewardBoost := 0.0
-	const runs = 1
+	const runs = 50
```

Comment:
we probably want to reset this for merge so that the test doesn't take 5min to run?

Reply:
yeah